storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -253,9 +253,6 @@ ulint xtrabackup_log_copy_interval = 1000;
 char *log_ignored_opt = NULL;
 
 /* === metadata of backup === */
-#define XTRABACKUP_METADATA_FILENAME "xtrabackup_checkpoints"
-static char metadata_type_str[30] = ""; /*[full-backuped|log-applied|
-                                        full-prepared|incremental]*/
```

Comment:
just removing static is fine? Any reason for moving it down?

Reply:
moved to xtrabackup.h